app/src/main/java/com/example/drawingApp/DrawingViewModel.kt

```diff
@@ -88,6 +88,16 @@ class DrawingViewModel : ViewModel() {
         updateViewState()
     }
 
+    fun openDeleteDialog() {
```

Comment:
As I left in Brandon's PR, the caller shouldn't know about what the _function_ of their UI interaction is. 

For example, I click a green button in the view. I shouldn't be calling `viewmodel.openDialog` I should be calling `viewmodel.greenButtonClicked`

Making your methods around `intented actions` rather than just `view events` leaks information to the viewer and makes your viewmodel harder to reuse or change the functionality. What if later you wanted slightly different behavior when the green button was clicked? You'd have to change the view layer when you should only need to change the viewmodel layer